sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -980,6 +981,7 @@ private void WakeLoop()
             // wake up the handler loop
             var handlerCts = Interlocked.Exchange(ref _handlerCts, new CancellationTokenSource());
             handlerCts.Cancel();
+            handlerCts.Dispose();
```

Comment:
I'm 78% certain you don't need to dispose after cancel.

Reply:
It made Rider happy

Reply:
https://stackoverflow.com/a/30855490/16249465

Reply:
Nice!